tests/test_diffeq/test_odefiltsmooth/test_odefiltsmooth.py

```diff
@@ -461,28 +442,9 @@ def test_filter_ivp_mat52_ekf(self):
             method="eks1",
         )
 
-    def test_filter_ivp_mat72_ukf(self):
-        """UKF requires some evaluation-variance to have a positive definite innovation
-        matrix, apparently."""
-        probsolve_ivp(
-            self.ivp,
-            atol=self.tol,
-            rtol=self.tol,
-            evlvar=0.01,
-            which_prior="matern72",
-            method="uks",
-        )
-
```

Comment:
I removed some of the high order Matern priors with UK observations, because the lack of evlvar parameter makes these numerically unstable.